pages/common/zstd.md

```diff
@@ -0,0 +1,23 @@
+# zstd
+
+> Compress or decompress files with Zstandard compression.
+
+- Compress a file into a new file with the .zst suffix:
+
+`zstd {{file}}`
+
+- Decompress a file:
+
+`zstd -d {{file}}.zstd`
+
+- Decompress to stdout:
+
+`zstd -dc {{file}}.zstd`
+
+- Compress a file, while specifing the compression level. 0 = Worst, 19 = Best (Default level is 3):
+
+`zstd -{{level}} {{file}}`
+
+- Compress a file with high compression levels, using more memory during compression and decompression. Max level is 22:
```

Comment:
Remove "during compression and decompression". Convert last sentence to inside parenthesis, as the description above.

Reply:
I've done as you said, but I think it adds value to mention that it uses more memory during decompression as well, as it's not obvious.
Would appreciate your feedback on that.

Reply:
Hmm .. that's right. You mean it will take more memory while decompressing the file as well right ? I agree that it is not obvious, but at the same time it makes the sentence a bit longer than I am comfortable with. @sbrl @pxgamer - thoughts ?

Reply:
I think it still doesn't clarify the fact that it will take more memory during decompression too.

Reply:
Ah, I see. Hrm. What about this then:

```
Use more memory (both when compressing and decompressing) to achieve a higher compression ratio:
```

Reply:
SGTM.